sipn_reanalysis_ingest/luigitasks/untar.py

```diff
@@ -57,6 +59,30 @@ def output(self):
         )
 
 
+class UntarCfsr1DayFile(UntarFileTask):
+    """Untar a daily CFSR tar file."""
+
+    window_start = luigi.DateParameter()
+    window_end = luigi.DateParameter()
+    product_type = luigi.EnumParameter(enum=CfsrGranuleProductType)
```

Comment:
The 1-daily data comes with forecast and analysis data in one tar, so we can remove this parameter.